serial_views.py

```diff
@@ -0,0 +1,153 @@
+# serialize data for all models
+from django.http import HttpResponse
+
+
+import WAM_APP_FRED.oep_models as oep_modles
+# from WAM_APP_FRED.db_sqla import *
+from shapely.wkb import loads as loadswkb
+from geojson import Feature, FeatureCollection, dumps
+from geoalchemy2 import functions
+import sqlahelper as sah
+from datetime import datetime
+
+# def serializer():
+#     """
+#     returns the queryed table id, geom as GEOJSON featureCollection
+#     serializer for table without relations
+#     :return: dict - geojson featureCollection
+#     """
+#
+#     #ToDO: after testing done change to input pram
+#     Session = sah.get_session()
+#     session = Session()
+#     ##############################################
+#
+#     features = []
+#
+#     for record in session.query(oep_modles.classes['Location']).limit(100):
+#         geometry = loads(str(record.point), True)
+#         propertys = record.id
+#         feature = Feature(id=record.id, geometry=geometry)
+#
+#         features.append(feature)
+#
+#     return dumps(FeatureCollection(features))
+
+class Serializer():
+    """
+    returns a query result containing a full record from OEP table as GEOJSON featureCollection.
+    All related tables are joined and the values are included as property within the GEOJSON.
+
+    :return: dict - geojson featureCollection
+    """
+
+    #ToDO: after testing done change to input pram
+    Session = sah.get_session()
+    session = Session()
+    ##############################################
+
+    def wseries_geometry_view(self):
+        """
+        returns a query result containing a full record from OEP table as GEOJSON featureCollection.
+        Just the geometry is included.
+        All related tables are joined and the values are included as property within the GEOJSON.
+
+        :return:
+        """
+
+        features = []
+
+        for record in Serializer.session.query(oep_modles.classes['Series'],
+                                               oep_modles.classes['Location'])\
+                                                .join(oep_modles.classes['Location']).limit(100000):
+
+            geometry = loadswkb(str(record.Series.location.point), True)
+            feature = Feature(id=record.Series.id, geometry=geometry)
+            features.append(feature)
+
+        return HttpResponse(dumps(FeatureCollection(features)), content_type="application/json")
+
+
+
+    def wseries_property_view(self):
```

Comment:
created different methods that return the geometry or the propertys for each database record in order to reduce the amount of data in a single GeoJSON.